server/mailet/integration-testing/src/main/java/org/apache/james/mailets/configuration/SmtpConfiguration.java

```diff
@@ -58,13 +59,15 @@ public static class Builder {
         private Optional<SMTPConfiguration.SenderVerificationMode> verifyIndentity;
         private Optional<Boolean> bracketEnforcement;
         private Optional<String> authorizedAddresses;
+        private Optional<Integer> maxRcptHandler;
```

Comment:
:-1: we shall configure this deirectly through the handle, not as a global server configuration

Reply:
I wonna write:

```
    <smtpserver enabled="true">
        <jmxName>smtpserver-global</jmxName>
        <bind>0.0.0.0:0</bind>
        <connectionBacklog>200</connectionBacklog>
        <tls socketTLS="false" startTLS="false">
            <keystore>file://conf/keystore</keystore>
            <secret>james72laBalle</secret>
            <provider>org.bouncycastle.jce.provider.BouncyCastleProvider</provider>
            <algorithm>SunX509</algorithm>
        </tls>
        <connectiontimeout>360</connectiontimeout>
        <connectionLimit>0</connectionLimit>
        <connectionLimitPerIP>0</connectionLimitPerIP>
        <authRequired>{{authRequired}}</authRequired>
        <auth>
            <requireSSL>false</requireSSL>
            <plainAuthEnabled>true</plainAuthEnabled>
        </auth>
        <verifyIdentity>{{verifyIdentity}}</verifyIdentity>
        <maxmessagesize>{{maxmessagesize}}</maxmessagesize>
        <addressBracketsEnforcement>{{bracketEnforcement}}</addressBracketsEnforcement>
        <smtpGreeting>Apache JAMES awesome SMTP Server</smtpGreeting>
        <handlerchain>
            <handler class="org.apache.james.smtpserver.fastfail.ValidRcptHandler"/>
            <handler class="org.apache.james.smtpserver.CoreCmdHandlerLoader"/>
            <handler class="org.apache.james.smtpserver.MaxRcptHander">
               <maxRcptCount>42</maxRcptCount> 
           </handler>
        </handlerchain>
        <gracefulShutdown>false</gracefulShutdown>
    </smtpserver>
```

CF DKIMHook